public/utils/contextMenu/getActions.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { i18n } from '@osd/i18n';
+import { EuiIconType } from '@elastic/eui/src/components/icon/icon';
+import { toMountPoint } from '../../../../../src/plugins/opensearch_dashboards_react/public';
+import { Action } from '../../../../../src/plugins/ui_actions/public';
+import { createADAction } from '../../action/ad_dashboard_action';
+import Container from '../../components/FeatureAnywhereContextMenu/Container';
+import DocumentationTitle from '../../components/FeatureAnywhereContextMenu/DocumentationTitle';
+
+// This is used to create all actions in the same context menu
+const grouping: Action['grouping'] = [
+  {
+    id: 'ad-dashboard-context-menu',
+    getDisplayName: () => 'Anomaly Detector',
+    getIconType: () => 'apmTrace',
```

Comment:
Can we extract `apmTrace` into a constant, maybe in `public/utils/constants`? This way it can be used in other components potentially, or if it changes in the future due to EUI->OUI migration we can easily update.

Reply:
sure, will do